can-attribute-encoder.js

```diff
@@ -50,7 +50,7 @@ var regexes = {
 	rightParens: /\)/g,
 	leftBrace: /\{/g,
 	rightBrace: /\}/g,
-	camelCase: /([a-z]|^)([A-Z])/g,
+	camelCase: /([a-z]|^)([0-9]|[A-Z])/g,
```

Comment:
I think this should be `/([a-z]|[0-9]|^)([A-Z])/g`. What you have still works, but it will unnecessarily match things like `foo1bar:to`, when nothing needs to be encoded in this case.

Reply:
Done!